pkg/modprovider/tfmodules.go

```diff
@@ -365,7 +365,7 @@ func inferModuleSchema(
 			TypeSpec:    variableType,
 		}
 
-		if variable.Default.IsNull() && !variable.Nullable {
+		if variable.Default.IsNull() && !(variable.NullableSet && variable.Nullable) {
```

Comment:
> Looks like lots of inputs from modules we test are now flagging a ton of their inputs as non-nullable

Do you know how the `Nullable` and `NullableSet` gets determined? This is one of the properties from the test failures that is being read as `required` now.

https://github.com/terraform-aws-modules/terraform-aws-s3-bucket/blob/0d781fbb515b1f64c46bfba04153bef091d50fc5/variables.tf#L133-L137

Reply:
I believe `NullableSet` means that the source code has an attribute `nullable = <true | false>` and then `Nullable` if the `nullable` is set to true. I will look again 😅